### PR TITLE
fix pushing to generators

### PR DESCRIPTION
**Pull request by thepureinx000**
I found some errors in createGenerator method.

### CODE REVIEW

**Comment by dscush** [comment]
Wow...I did _not_ test my most recent commit very well! Thanks for fixing it. This is my first time creating a React project from the beginning, so the convention improvements are helpful too. There are just two little details I think should be changed (I'll leave comments on the lines) and then I'll merge it in when those are fixed. Thanks again!

File: src/App.jsx [hunk 1]
```diff
@@ -1,36 +1,36 @@
 import React, { Component } from 'react';
-import './App.css';
+import './App.scss';
```
Comment (dscush) [inline src/App.jsx]: The way create-react-app set things up, it automatically compiles (and watches and re-compiles) Sass files into CSS files, so this should remain an import of the compiled CSS.

File: src/index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import './index.scss';
```
Comment (dscush) [inline src/index.js]: Same as in App.jsx - see above.

**Review by dscush — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (thepureinx000) [reply src/index.js]: exactly!

**Review by thepureinx000 — COMMENTED** [review]
Fixed

**Review by dscush — APPROVED** [review]

**Pull request merged**